src/handlers/Marriage.cpp

```diff
@@ -82,14 +82,16 @@ void Marriage_Request(Character *character, PacketReader &reader)
 				return;
 			}
 
-			character->DelItem(1, marriage_price);
```

Comment:
Was this line responsible for deducting the cost of a marriage from a character’s balance? If so, it looks like marriage would now be free.

Reply:
Oops. Thanks, that should be fixed now.